src/modules/company/dto/ReservationOption.dto.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { BillingType } from '@prisma/client'
+import { IsDefined, IsEnum, IsNumber, IsString } from 'class-validator'
+
+import { AgeGroupPriceDTO } from './AdditionalAgeGroupsPrices.dto'
+import { ReservationOptionHousingUnitTypeDTO } from './ReservationOptionHousingUnitType.dto'
+
+export class ReservationOptionDTO {
+    @ApiProperty({ example: 'Standard Reservation', type: String })
+    @IsDefined()
+    @IsString()
+    name!: string
+
+    @ApiProperty({ enum: BillingType, example: BillingType.DAILY })
```

Comment:
```suggestion
    @ApiProperty({ enum: BillingType, enumName: 'BillingType' example: BillingType.DAILY })
```

Vamos começar a colocar o nome do enum, assim ele reutiliza o mesmo enum